pupil-api/docker-start.sh

```diff
@@ -10,4 +10,8 @@ then
   yarn migrate-sql
 fi
 
+echo "Starting SSH..."
+service ssh start
```

Comment:
Again, I would start the `ssh` service early.  I have no idea why `docker-start` is running migrations from this container.  I think they can be removed.